packages/config-driver-ssm/readme.md

```diff
@@ -0,0 +1,72 @@
+# Etrigan AWS SSM Parameter Store Config Driver
+
+Allows Etrigan config to load values from AWS Parameter store
+
+## Example
+
+```ts
+import { loadConfig, getConfigRecords, registerDriver } from '@etrigan/config'
+import { parameterStoreConfigDriver } from '@etrigan/config-driver-ssm'
+
+registerDriver(parameterStoreConfigDriver)
+
+const config = loadConfig<Config>({
+    values: await getConfigRecords('ssm://my-app/dev/ region=ap-southeast-2'),
+    validateConfig: {
+        val: 'required-string',
+        secret: 'required-string',
+    },
+})
+```
+
+## Creating keys
+
+Configuration parameters for all environments and applications can be managed using the EC2 Management Console, Systems Manager Console, aws ssm CLI interface or this little batch import/export script I through together: param-sync.
```

Comment:
I'm not sure that's the "through" you're looking for

Reply:
Nice catch.